yaksh/forms.py

```diff
@@ -173,6 +173,12 @@ def clean(self):
         return user
 
 
+class ExerciseForm(forms.ModelForm):
+    class Meta:
+        model = Quiz
+        fields = ['description']
```

Comment:
I feel Exercise form should also have `allow students to view answer paper` if suppose students want to view their answer papers.

Reply:
Okay providing the option